app/src/main/java/cz/quanti/razym/rocketapp/domain/RocketsRepo.kt

```diff
@@ -0,0 +1,5 @@
+package cz.quanti.razym.rocketapp.domain
+
+interface RocketsRepo {
```

Comment:
mozna jsem to blbe napsal na ten papir, ale pouzivame cele slovo Repository pro interface i implementaci